src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -771,8 +775,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                 const auto cursorPosition = point.Position();
                 _SetEndSelectionPointAtCursor(cursorPosition);
 
-                const double cursorBelowBottomDist = cursorPosition.Y - _swapChainPanel.Margin().Top - _swapChainPanel.ActualHeight();
-                const double cursorAboveTopDist = -1 * cursorPosition.Y + _swapChainPanel.Margin().Top;
+                const double cursorBelowBottomDist = cursorPosition.Y - _padding.Top - _swapChainPanel.ActualHeight();

```

Comment:
Is there ever a time that `_padding != _swapChainPanel.Margin()`?

Reply:
No, there shouldn't be.

Reply:
If there's not, then I think I'd prefer if we kept it in `_swapChainPanel.Margin()`. We've been burned before by keeping "redundant" data around that ends up getting out of sync.